styles/variables-base.less

```diff
@@ -166,19 +175,22 @@
 // Icon Sizes
 // ---------------------------------------
 @agate-icon-size: 48px;
-@agate-icon-small-size: 36px;
-@agate-icon-huge-size: 64px;
 @agate-icon-font-size: (@agate-icon-size * 2);
-@agate-icon-small-font-size: (@agate-icon-small-size * 2);
-@agate-icon-huge-font-size: 400px;
-@agate-icon-button-size: @agate-button-height;
-@agate-icon-button-small-size: @agate-button-small-height;
 @agate-icon-sprite-size: @agate-icon-size;
-@agate-icon-sprite-small-size: @agate-icon-small-size;
-@agate-icon-sprite-huge-size: @agate-icon-huge-size;
+@agate-icon-button-size: @agate-button-height;
 @agate-icon-margin: @agate-component-spacing;
 @agate-icon-letter-spacing: inherit;
 @agate-icon-text-indent: initial;
+@agate-icon-smallest-size: 21px;
+@agate-icon-smallest-font-size: (@agate-icon-smallest-size * 2);
+@agate-icon-sprite-smallest-size: @agate-icon-smallest-size;
+@agate-icon-small-size: 36px;
+@agate-icon-small-font-size: (@agate-icon-small-size * 2);
+@agate-icon-sprite-small-size: @agate-icon-small-size;
+@agate-icon-button-small-size: @agate-button-small-height;
```

Comment:
Do these variables: `@agate-icon-button-small-size` make sense to retain or should they be renamed to fit into the Button section? Where are they used?